src/main/java/org/project36/qualopt/service/StudyService.java

```diff
@@ -75,4 +102,31 @@ protected PasswordAuthentication getPasswordAuthentication() {
                 }
             });
     }
+    /**
+     * Method is used to create a schedule for a job: invitation email to be registered
+     * with the the Quartz Scheduler.
+     */
+    private void scheduleEmailJob(MimeMessage message, int delay) throws SchedulerException {
+        
+        SchedulerFactory schedFact = new StdSchedulerFactory();
+        Scheduler sched = schedFact.getScheduler();
+
+        sched.start();
+        JobDataMap messageMap = new JobDataMap();
+        messageMap.put("message", message);
+
+        JobDetail job = newJob(EmailScheduler.class)
+            .withIdentity("myJob", "group1")
+            .usingJobData(messageMap)
+            .build();
+
+        // Trigger the job to run after a certain delay
+        Trigger trigger = newTrigger().withIdentity("myTrigger", "group1")
+        .startAt(futureDate(delay,IntervalUnit.SECOND))
```

Comment:
nitpicking on the indentation here.... :( sorry.

Reply:
this has been fixed now :)